style.css

```diff
@@ -0,0 +1,717 @@
+
+.navigation__search {
+  width: 9%;
+  padding: 0;
+  border: none;
+  background: url(images/search.png) no-repeat 100%;
+}
+
+.navigation__submenu {
+  display: flex;
+  justify-content: space-around;
+  flex: 1;
+  margin: 2px 0.9% 0 0;
+}
+
+.navigation__img {
+  margin-right: 4.4%;
+  padding-top: 1px;
+}
+
+.navigation__link span {
+  display: inline-block;
+  border: none;
+  color: #FFF;
+  font-family: Gotham Pro, Arial, serif;
+  font-size: 10px;
+  font-weight: bold;
+  text-align: center;
+  line-height: 14.92px;
+  background: #000;
+  width: 14.92px;
+  height: 14.92px;
+  border-radius: 50%;
+  padding: 0;
+}
+
+.navigation__us {
+  width: 20px;
+  height: 20px;
+}
+
+.navigation__burger {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: bold;
+  color: #000;
+}
+
+.video__main, 
+.stories__text {
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.catalog__wrapper {
+  margin-bottom: 49px;
+  padding: 89px 0 0 60px;
+  width: 100%;
+  height: 280px;
+  background: url(images/background.jpg) no-repeat;
+  background-size: cover;
+  color: #FFF;
+}
+
+.catalog__title {
+  margin: 0;
+  padding-bottom: 12px;
+  width: 25%;
+  font-size: 30px;
+  line-height: 29px;
+}
+
+.catalog__shop {
+  margin: 0;
+}
+
+.catalog__shop a {
+  line-height: 11px;
+  color: #FFF;
+}
+
+a:hover { color: #F33; }
+
+a img:hover { background: #F33; }
+
+button:hover { 
+  cursor: pointer;
+  opacity: .5;
+}
+
+.catalog__arrow {
+  padding-left: 3px;
+  width: 4.8%;
+}
+
+.catalog__subtitle {
+  margin-bottom: 18px;
+  text-align: center;
+  font-size: 22px;
+}
+
+.catalog__menu {
+  display: flex;
+  justify-content: space-around;
+  flex-flow: row wrap;
+  margin-bottom: 26px;
+}
+
+.catalog__item {
+  width: 20%;
+  height: 261px;
+  min-width: 205px;
+}
+
+.catalog__item_1 {
+  margin-right: 3.5%;
+}
+
+.catalog__item_1:before{
+  position: relative;
+  content: "";
+  background: #000;
+  opacity: 0.2;
+  width: 1px;
+  height: 100.18px;
+  float: right;
+  top: 42px;
+  left: 15%;
+}
+
+.catalog__img {
+  margin-left: 5%;
+  margin-bottom: 23px;
+}
+
+.catalog__link {
+  display: block;
+  font-weight: normal;
+  line-height: 16px;
+}
+
+.catalog__wrap {
+  display: flex;
+  justify-content: space-between;
+}
+
+.catalog__price {
+  display: inline;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+  width: 30%;
+}
+
+.catalog__scale {
+  display: inline-flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 25%;
+}
+
+.catalog button {
+  border: none;
+  padding: 0;
+  border-radius: 50%;
+}
+
+.catalog__round_grey {
+  background: #C4C4C4;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_black {
+  background: #0E1211;
+  width: 9px;
+  height: 9px;
+}
+
+.catalog__round_red {
+  background: #B62E3C;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_yellow {
+  background: #FDB431;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_peach {
+  background: #E29D69;
```

Comment:
Codacy found an issue: [Expected "#E29D69" to be "#e29d69" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)